app/src/main/java/mozilla/lockbox/LockboxAutofillService.kt

```diff
@@ -72,10 +73,12 @@ class LockboxAutofillService(
     }
 
     override fun onFillRequest(request: FillRequest, cancellationSignal: CancellationSignal, callback: FillCallback) {
+        touchRecentlyUsedDatasets()
+
         val structure = request.fillContexts.last().structure
         val activityPackageName = structure.activityComponent.packageName
         if (this.packageName == activityPackageName) {
-            callback.onSuccess(null)
+            callback.onFailure(null)
```

Comment:
Why are these being changed from success -> failure?

Reply:
It's a bit debatable whether this is a failure or success.

But from the look of the event history, not having any fillable things is likely to be a failure.